cartography/intel/aws/ec2/images.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+from botocore.exceptions import ClientError
+
+from .util import get_botocore_config
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_images(boto3_session: boto3.session.Session, region: str) -> List[Dict]:
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    try:
+        images = client.describe_images()['Images']
+    except ClientError as e:
+        logger.warning(f"Failed retrieve images for region - {region}. Error - {e}")
+        raise
+    return images
+
+
+@timeit
+def load_images(
+        neo4j_session: neo4j.Session, data: List[Dict], region: str, current_aws_account_id: str, update_tag: int,
+) -> None:
+    ingest_images = """
+    UNWIND {images_list} as image
+        MERGE (i:EC2Image{id: image.ImageId})
```

Comment:
Is the ImageId guaranteed to be unique across all AWS like an ARN?

Reply:
If ImageId is only unique per region, let's set `i.id` to be `{ImageId}|{region}` and set `i.imageid = {ImageId}`. Also add an index for `i.imageid`.

Reply:
cc: @mpurusottamc

Reply:
@achantavy AMI ID are unique per region. So, we will follow your suggestion of appending region to the id.
[Finding an AMI](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/finding-an-ami.html#finding-an-ami-console)
Thanks @kedarghule

Reply:
@achantavy and @mpurusottamc  Changes made and committed 🚀